statemachine/statemachine.py

```diff
@@ -84,6 +84,11 @@ def __init__(
         self._setup(initial_transition)
         self._activate_initial_state(initial_transition)
 
+
+    def __init_subclass__(cls, strict_states: bool = False):
```

Comment:
**question:** Why this change was required?

Reply:
This is the method that gets called first when using class parameter syntax.

```python
class MyStateMachine(StateMachine, strict_states=True): # __init__subclass__ called here
    ...
```